extension/src/test/suite/setup/index.test.ts

```diff
@@ -850,49 +854,100 @@ suite('Setup Test Suite', () => {
       ).to.be.calledWithExactly('setContext', 'dvc.cli.incompatible', false)
     })
 
-    it('should handle a message from the webview to open Studio', async () => {
-      const { mockOpenExternal, setup, urlOpenedEvent } = buildSetup({
+    it('should handle a message from the webview to request a token from studio', async () => {
+      const { setup, mockFetch, studio } = buildSetup({
         disposer: disposable
       })
 
+      const mockConfig = stub(DvcConfig.prototype, 'config')
+      mockConfig.resolves('')
       const webview = await setup.showWebview()
       await webview.isReady()
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      stub(Setup.prototype as any, 'getCliCompatible').returns(true)
```

Comment:
Why is this private in the first place?

```
private getCliCompatible() {
    return this.cliCompatible
  }
```
That does not seem to have any use since it's accessing a local variable. Is this code for tests only? Is there no other way to change the value? Why not set it public then? cc. @mattseddon (I think you're the one that added it)